empress/recon_vis/utils.py

```diff
@@ -3,17 +3,75 @@
 Utilities related to conversion between data types
 """
 
-from collections import OrderedDict 
 from typing import Dict, Tuple, List
-from empress.recon_vis.recon import MappingNode, Reconciliation
+from empress.recon_vis.recon import MappingNode, ReconGraph, Reconciliation
 from empress.recon_vis.recon import Cospeciation, Duplication, Transfer, Loss, TipTip
-from empress.recon_vis import tree
+import empress.recon_vis.tree as tree
```

Comment:
Use `from empress.recon_vis import tree` over `import empress.recon_vis.tree as tree`. It does the same thing without mentioning `tree` twice.

Reply:
fixed